TypeCobol.Analysis/Dfa/ValueOrigin.cs

```diff
@@ -137,6 +136,29 @@ void ProcessSetStatement(SetStatement set)
                             break;
                     }
                 }
+
+                void ProcessDataDefinition(DataDefinition dataDef)
+                {
+                    VariableSymbol vs = (VariableSymbol)dataDef.SemanticData;
+                    System.Diagnostics.Debug.Assert(vs.Value != null);
+                    //(2.1.1.3.1) Take the initial value and add it to the origins path.
+                    if (vs.Value is Value value)
+                    {                            
+                        var origin = new ValueOrigin(vs)
+                        {
+                            Value = value
+                        };
+                        result.Origins.Add(origin);
+                    }
+                    else if (vs.Value is Value[] values && values.Length > 0)
+                    {
+                        var origin = new ValueOrigin(vs)
+                        {
+                            Value = values[0]
+                        };
+                        result.Origins.Add(origin);
+                    };                        
```

Comment:
I am not so sure about this part because I've tested this source:
```cobol
       IDENTIFICATION DIVISION.
       PROGRAM-ID. TCOMFL06.
       DATA DIVISION.
       WORKING-STORAGE SECTION.
       01 somecondition PIC X(5).
          88 cond1  VALUE 'A'.
          88 cond2 VALUES 'B' 'C' 'D'.
          88 cond3  VALUE 'E'.
       PROCEDURE DIVISION.
           DISPLAY "somecondition: " somecondition
           evaluate true
           when cond1 display "cond1"
           when cond2 display "cond2"
           when cond3 display "cond3"
           when other display "other"
           end-evaluate
           GOBACK
           .
       END PROGRAM TCOMFL06.
```

and the output is:
```
somecondition:      
other

```

So it looks like a data condition has no initial value. I think we should remove this `else if` block and also remove the `case CodeElementType.DataConditionEntry` in parent switch.

Reply:
@fm-117 Hum... I had also a doubt about conditions, but as Conditions are also DataDefinition instances, a treat them  with the same logic, but I am curious to know the context of your test: whar is the test which give you such result as output?
Anyway ZcallPgmReport does a special treatment on level 88 so in what context do you use your test?

Reply:
Ok, if as you have tested it is a nonsense, to consider 88 value as read/write value ok.

Reply:
done in 37ed136